### DISCUSSION TITLE
Feature Request: hold action for Tile cards

### DISCUSSION THREAD

**Discussion by jorgwolski** [home-assistant/frontend#19847] [Dashboard] [answered]
It would be nice to have a hold action like the Button cards.
Especially for mobile app users it is sometimes useful not to trigger an action on an accidental tap.

**Comment by jorgwolski** [comment]
A little addition:
A hold and double-tap action is already possible via the YAML configuration (Missed that unill now).
An option via UI would still be a good addition.

**Reply by karwosts** [reply, marked as answer]
I did see this comment from HA design team, which probably explain why it's hidden in the UI. 

> discussed with UX team and we don't want to encourage users to use hold and double tap action because it can be difficult to perform on some device. However, we're not opposed to bring this feature as a YAML only feature for now.

https://github.com/home-assistant/frontend/pull/18345#issuecomment-1831591770
